parm/convection_permitting/namelist.atmosphere

```diff
@@ -53,28 +53,14 @@
     config_radtlw_interval = '00:15:00'
     config_radtsw_interval = '00:15:00'
     config_bucket_update = 'none'
-    config_physics_suite = '${physics_suite}'
-    config_microp_scheme = 'mp_thompson'
-    config_thompson_hail_aware = true
-    config_convection_scheme = 'off'
+    ! config_physics_suite = '${physics_suite}'
+    config_physics_suite = 'hrrrv5'
+    config_tempo_aerosolaware = .true.
+    config_tempo_hailaware = .true.
     config_gwdo_scheme = 'bl_ugwp_gwdo'
     config_lsm_scheme    = 'sf_ruc'
-    ! config_sfclayer_scheme   = 'sf_monin_obukhov'
-    ! config_pbl_scheme        = 'bl_ysu'
     num_soil_layers      = 9
-    config_mynn_cloudpdf = 2
-    config_mynn_mixlength= 1
-    config_mynn_edmf = 1
-    config_mynn_edmf_mom = 1
-    config_mynn_edmf_tke = 0
-    config_mynn_edmf_output = 0
-    config_mynn_closure = 2.6
-    config_mynn_mixscalars = 1
-    config_mynn_mixclouds = 1
-    config_mynn_mixqt = 0
-    config_mynn_tkeadvect = .false.
-    config_mynn_tkebudget = 0
-    config_radt_cld_scheme='cld_fraction_mynn'
+    config_pbl_scheme = 'bl_mynnedmf'
 /
```

Comment:
so many model options are changed. But those are used by conus3km real-time runs. Need to figure out how to set conus 12km separately.

Reply:
After talk with Joe, I found that the most options for mynn on old namelist are not needed if we use the new version of the model. We can use separate namelist for now but both (3km, 12km) should be close many be combined in the future.